app/controllers/articles_controller.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 class ArticlesController < ApplicationController
+  # rubocop:disable Metrics/AbcSize
   def index
     @articles = if !params[:query] || params[:query].empty?
-                  Article.published.in_language(extract_locale).sorted_desc
+                  Article.includes(:article_tags, :tags).published.in_language(extract_locale).sorted_desc
```

Comment:
Move this `Article.includes(:article_tags, :tags).published.in_language(extract_locale)` to another method for examples `def articles`

Reply:
And after that check do you really need rubocop disable

Reply:
And you can reuse the sorting for search result